landing-page/content/common/how-to-release.md

```diff
@@ -235,6 +265,8 @@ svn add apache-iceberg-<VERSION>
 svn ci -m 'Iceberg: Add release <VERSION>'
 ```
 
+Note, the above step requires PMC privileges to execute.
```

Comment:
I think we only need committer level privilege?

Reply:
Also we can use `!!!Note` syntax to format the text

Reply:
In fact the step to push the release to SVN requires PMC privileges. I was working together with @Fokko on this and even him didn't have enough permissions so we had to include a PMC member to execute this.

I tried this "!!! Note" syntax, but honestly don't get the point :) This just writes the same in the page. Do I miss something?